formatters/cloudevents/docs.go

```diff
@@ -0,0 +1,6 @@
+// cloudevents package includes a formatting Node which will transform and
```

Comment:
Hum... `cloudevents` is the package name so I started the package godoc with `cloudevents`.  I don't think there's anyway to avoid this unless I rename the package.